### PR TITLE
Testing CI CD print on commit

### PR DESCRIPTION
**Pull request by codeAesthetic**

### CODE REVIEW

File: src/Home/Home.tsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Link, RouteComponentProps } from "react-router-dom";
 import { IParams } from "src/common/interface";
 
 interface IPage {
-  name: string
+  name: string;
```
Comment (chirgjn) [inline src/Home/Home.tsx:7]: Unrelated change

File: .github/workflows/integration.yml [hunk 2]
```diff
@@ -34,13 +34,13 @@ jobs:
 
       - name: Test linting
         run: yarn lint  
-
+
```
Comment (chirgjn) [inline .github/workflows/integration.yml:37]: Unrelated change

**Review by chirgjn — CHANGES_REQUESTED** [review]
Please update the PR title and description to explain the intent

**Comment by codeAesthetic** [comment]
This is was testing PR ignore
